notebooks/uncertainty_aware_models/solutions/uncertainty_quantification_with_sngp_bert.ipynb

```diff
@@ -0,0 +1,1092 @@
+{
```

Comment:
<p>It seems the <code>ClassificationHead()</code> layer is a simple stacks of Dense -&gt; Dropout -&gt; Dense layers. Maybe it is worth mentioning it to avoid unnecessary black-box?</p><p>https://github.com/tensorflow/models/blob/8c7699ceacfa5bec10bccc9879705d73b0d8db85/official/nlp/modeling/layers/cls_head.py#L25</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done!</p>